data-prepper-core/src/test/java/org/opensearch/dataprepper/peerforwarder/PeerForwarderConfigurationTest.java

```diff
@@ -144,12 +145,21 @@ void test_with_acm_should_create_PeerForwarderConfiguration_object_even_with_nul
         assertThat(peerForwarderConfiguration.isUseAcmCertificateForSsl(), equalTo(true));
     }
 
+    @Test
+    void test_cert_paths_with_ssl() throws IOException {
+        final PeerForwarderConfiguration peerForwarderConfiguration = makeConfig(TestDataProvider.INVALID_PEER_FORWARDER_WITH_SSL_CONFIG_FILE);
+
+        assertThat(peerForwarderConfiguration.isSsl(), equalTo(true));
+        assertThat(peerForwarderConfiguration.getSslCertificateFile(), equalTo("invalid_path"));
+        assertThat(peerForwarderConfiguration.getSslKeyFile(), equalTo(DEFAULT_PRIVATE_KEY_FILE_PATH));
+    }
+
     @ParameterizedTest
     @ValueSource(strings = {
             TestDataProvider.INVALID_PEER_FORWARDER_WITH_PORT_CONFIG_FILE,
             TestDataProvider.INVALID_PEER_FORWARDER_WITH_THREAD_COUNT_CONFIG_FILE,
             TestDataProvider.INVALID_PEER_FORWARDER_WITH_CONNECTION_CONFIG_FILE,
-            TestDataProvider.INVALID_PEER_FORWARDER_WITH_SSL_CONFIG_FILE,
+//            TestDataProvider.INVALID_PEER_FORWARDER_WITH_SSL_CONFIG_FILE
```

Comment:
Is this test still valid since the file doesn't exist? Or does the validation not yet happen?